Automattic/wc-calypso-bridge
# Free Trial: Restrict Extensions Marketplace

### Changes proposed in this Pull Request:



Closes #965 






### How to test the changes in this Pull Request:




1. Navigate to **Extensions > Discover** while in ecom free trial site.
2. Notice the landing page instead of the WC Marketplace.
3. Navigate to an Ecommerce site.
4. Ensure that the **Extensions > Discover** shows the marketplace.



### Other information:




### FOR PR REVIEWER ONLY:
